template-for-repository/proofs/Makefile.common

```diff
@@ -246,45 +247,10 @@ ARPA_COMP_CMDS ?= $(ARPA_BLDDIR)/compile_commands.json
 SPACE :=$() $()
 COMMA :=,
 
-################################################################
-# Useful macros for running commands
-
-#1: command, 2: flags, 3: log file
-DO_AND_LOG_COMMAND = $(1) $(2) 2>&1 | tee $(3); exit $${PIPESTATUS[0]}
-
-#1: command, 2: flags, 3: log file
-
-# CBMC uses the special error-code 10 to signify that it detected an
-# assertion violation.	Continue the build so we can output the trace.
-DO_AND_LOG_IGNORING_ERROR_10 =	$(1) $(2) 2>&1 | tee $(3); if [ $${PIPESTATUS[0]} -ne 10 ]; then exit $${PIPESTATUS[0]}; fi
-
-#1: flags, 2: source, 3: target
-DO_GOTO_ANALYZER = $(call DO_AND_LOG_COMMAND,$(GOTO_ANALYZER),$(CBMC_VERBOSITY) $(1) $(2) $(3), $(call LOG_FROM_ENTRY,$(3)))
-
-#1: flags, 2: source, 3: target
-DO_GOTO_CC =  $(call DO_AND_LOG_COMMAND,$(GOTO_CC),$(CBMC_VERBOSITY) $(1) $(2) -o $(3), $(call LOG_FROM_ENTRY,$(3)))
-
-#1: flags, 2: source, 3: target
-DO_GOTO_INSTRUMENT = $(call DO_AND_LOG_COMMAND,$(GOTO_INSTRUMENT),$(CBMC_VERBOSITY) $(1) $(2) $(3), $(call LOG_FROM_ENTRY,$(3)))
-
-#1: flags, 2: source, 3: logfile
-DO_CBMC =  $(call DO_AND_LOG_IGNORING_ERROR_10,$(CBMC),$(CBMC_VERBOSITY) --flush $(1) $(2), $(3))
-
-#1: message 2: source 3: dest
-DO_NOOP_COPY = cp $(2) $(3); echo $(1) | tee $(call LOG_FROM_ENTRY,$(3))
-
-################################################################
-# Useful macros translating filenames
-
-C_FROM_GOTO = $(patsubst $(GOTODIR)%,$(SRCDIR)%,$(1:.goto=.c))
-GOTO_FROM_C = $(patsubst $(SRCDIR)%,$(GOTODIR)%,$(1:.c=.goto))
-LOG_FROM_ENTRY = $(LOGDIR)/$(notdir $(1:.goto=.txt))
-LOG_FROM_GOTO = $(patsubst $(GOTODIR)%,$(LOGDIR)%,$(1:.goto=.txt))
-
 ################################################################
 # Set C compiler defines
 
-CBMCFLAGS +=  --object-bits $(CBMC_OBJECT_BITS)
+CBMCFLAGS += --object-bits $(CBMC_OBJECT_BITS) --flush
```

Comment:
[Unrelated to changes in PR]

May be use this section only for compiler defines, and move line 253 next to line 113 to keep it together with other `CBMCFLAGS` stuff?